#001 Testing Travis

davidfrank96 opened the pull request:
What does this PR do?
* Testing  Travis setup
Description of Task to be completed?
- included a test dev script for testing locally
- installed babel register for testing
Any background context you want to provide?
Ensure that the repo is properly cloned locally
What are the relevant pivotal tracker stories?
#001
Screenshots (if appropriate)
None
Questions:

Stylll commented on package.json:
```diff
@@ -16,7 +16,8 @@
     "generate-lcov": "nyc report --reporter=text-lcov > lcov.info",
-    "test": "cross-env NODE_ENV=test mocha --require @babel/register src/test/**.test.js --timeout 20000 --exit"
+    "test": "cross-env NODE_ENV=test mocha --require @babel/register src/test/**.test.js --timeout 20000 --exit",
+    "testdev": " mocha --require @babel/register src/test/**.test.js --timeout 20000 --exit"
```
@davidfrank96 We don't need this script for `testdev`. Every test should be run in the test environment. We normally won't want to update the development database with test records.

Stylll reviewed (commented):
This PR may not be necessary. The changes therein are not required. You can close this PR.

davidfrank96 committed:
Chore(Testing Travis)
- included a testdev script for testing locally
- installed babel register for testing
[Finishes #001]
